ui-kit/HorizontalHighlightCard/HorizontalHighlightCard.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { systemPropTypes } from 'ui-kit';
+import Styled from 'ui-kit/DefaultCard';
+
+const HorizontalHighlightCard = (props = {}) => {
```

Comment:
That's a good point, that was something I wanted to ask you about and I agree we do need to keep it consistent. In order to keep it consistent with our other projects, I would prefer to use the `const` arrow func format since that's what we have been using across our api, mobile app, and current web app. But let me know what you think!

Reply:
Your call! This project is using the `function` pattern, but you can leave this and slowly migrate things over to match the other codebases 👍